lipscore.php

```diff
@@ -297,6 +297,10 @@ public function hooks() {
 						10,
 						2
 				);
+
+				$shortcode_manager = Lipscore_Shortcode_Manager::get_instance();
+		
+				add_action( 'init', [ $shortcode_manager, 'register_shortcodes' ] );
```

Comment:
I would remove the empty line between these two rows to reflect logical structure more clear (the rows are related).